app/api/datastore/entities/maat/application.rb

```diff
@@ -33,7 +33,8 @@ def provider_details
 
         def case_details
           case_details = application_value('case_details')
-          case_details['offence_class'] = nil
+          case_details['offence_class'] =
+            ::Helpers::OffenceClassCalculator.new(offences: case_details['offences']).offence_class
```

Comment:
I'm okay with this.

Reply:
Oh I think it may be Rails, which has its own Helpers module exposed.
Perhaps best to go with something else to avoid confusion? Maybe `Util` or `Utils` ...
But not something critical 👍

Reply:
Ah ok I didn't consider that. I'm ok with it too but just renamed for simplicity